src/common/contexts/TokenInformationContext/TokenInformationContext.tsx

```diff
@@ -3,12 +3,16 @@ import { TitleEscrow } from "@govtechsg/token-registry/types/TitleEscrow";
 import React, { createContext, useContext, useEffect, useState, useCallback } from "react";
 import { useTitleEscrowContract } from "../../hooks/useTitleEscrowContract";
 import { useProviderContext } from "../provider";
+import { useSupportsInterface } from "../../hooks/useSupportsInterface";
+
+export type TokenOwnerType = "TitleEscrow" | "Contract" | "Wallet";
```

Comment:
Actually `Wallet` is not technically correct, it can be (although unlikely) a contract that was yet to deployed. Both CREATE1 and CREATE2 can do this. 

CREATE1 can use nonce forecasting
CREATE2 can use counterfactual deployment to reserve an address

Reply:
we can't really detect that though, the only thing we can determine is that there is no contract at that address - so the question is whether "Wallet" is suitable to capture that or not?